addons/microsoft_calendar/models/calendar.py

```diff
@@ -130,18 +184,21 @@ def _microsoft_to_odoo_values(self, microsoft_event, default_reminders=(), defau
         return values
 
     @api.model
-    def _microsoft_to_odoo_recurrence_values(self, microsoft_event, default_reminders=(), values={}):
+    def _microsoft_to_odoo_recurrence_values(self, microsoft_event, default_values=None):
```

Comment:
Normally, we should not change the signature in stable version.
If we provide new parameter with default values, it's less problematic. Could you keep the default_reminders parameter even if you don't use it and add a comment to remove it in master?
@tivisse What do you think?

Reply:
up

Reply:
Hum, it should be ok, but as I did all the modifications in a dedicated commit to ease the review, it seems that this new commit is not taken into account to refresh the blocks of code in the review :thinking:

Reply:
Indeed. Thanks :-).